apb_devel/index.adoc

```diff
@@ -0,0 +1,201 @@
+[[apb-devel-intro]]
+= Introduction
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[apb-devel-intro-about]]
+== About This Guide
+
+This guide outlines the design concepts and workflow of
+link:https://github.com/ansibleplaybookbundle/ansible-playbook-bundle[Ansible Playbook Bundles (APBs)], shows how to install and use the `apb` CLI tooling,
+and provides a tutorial and reference material on writing your own APBs.
+
+[[apb-devel-intro-design]]
+== Design Overview
+
+An APB is a lightweight application definition that borrows several concepts
+from the link:https://github.com/projectatomic/nulecule[Nulecule] and
+link:http://www.projectatomic.io/docs/atomicapp/[Atomicapp] projects, namely the
+concept of a short-lived container with the sole purpose of orchestrating the
+deployment of the intended application. For the case of APBs, this short-lived
+container is the APB itself: a container with an
+link:https://www.ansible.com/[Ansible] runtime environment plus any files
+required to assist in orchestration, such as playbooks, roles, and extra
+dependencies.
+
+The
+xref:../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[OpenShift Ansible broker] (OAB) is an implementation of the Open Service Broker (OSB) API
+that manages applications defined by APBs. The OAB is supported and deployed by
+default starting in {product-title} 3.7.
+
+Specification of an APB is intended to be lightweight, consisting of several
+named playbooks and a metadata file to capture information such as parameters to
+pass into the application.
+
+[[apb-devel-intro-workflow]]
+== Workflow
+
+The APB workflow is broken up into the following steps:
+
+. Preparation
+.. APB initialization
+.. APB spec file
```

Comment:
Maybe spell it out in the first instance? Only if you think it's helpful: specification (spec) file